google/cloud/firestore_v1/vector.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List, Union
+
+
+class Vector:
+    r"""A class to represent Firestore Vector in python.
+
+    Underline it'll be converted to a map representation in Firestore API.
+    """
+
+    value: List[float] = []
```

Comment:
Also, should value be private?

Reply:
> Also, should value be private?

Yes. It should be private. Updated. (assuming mutable/immutable should be less a concern if private)

Reply:
I would still suggest defaulting to a tuple instead of a list here, since we don't expect the values to change. It should be more efficient, and safer.

Reply:
Good idea. Done